src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -294,13 +294,28 @@ foam.CLASS({
       code: async function() {
         var el    = await this.parentNode.el();
 
+        if ( ! this.data.currentWizardlet ) {
+          console.warn('setScrollPos() failed; no current wizardlet', {
+            data: this.data
+          });
+          return;
```

Comment:
Is this a fatal error for the wizard? Should be be throwing an error instead of just returning?

Reply:
The important thing is that it shows up in the console, but my thinking was breaking execution here doesn't make sense. If I remember correctly, this only happens when all the wizardlets have `isAvailable` as `false`, which should be supported but could be supported better.